examples/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ add_subdirectory(useCases)
 add_subdirectory(inlineMWE)
 add_subdirectory(plugins)
 
+if(Kokkos_ROOT)
```

Comment:
This will never be defined, you might want to include the kokkos directory anyways and in there find_package(kokkos...) and if found add_executable()

Reply:
I changed the way I am looking for Kokkos.

Reply:
Yes this is resolved, still we are having build errors building this, we need to resolve that first before merging

Reply:
https://github.com/ornladios/ADIOS2/runs/8029236659?check_suite_focus=true

Reply:
I am not seeing the error in the link you sent me. The fix you proposed seem to fail. I am not sure how I can debug something I cannot reproduce. Is this a CI issue or with the code?